packages/meditrak-server/src/apiV2/import/importEntities/updateCountryEntities.js

```diff
@@ -147,7 +143,7 @@ export async function updateCountryEntities(
       { code },
       {
         name,
-        parent_id: parentEntity.id,
+        parent_id: parentEntity ? parentEntity.id : null,
```

Comment:
A bit weird but using parentEntity?.id didn't update parent_id to null whereas using a trinary statement does. Couuldn't work out why.

Reply:
`undefined` get stripped before making it to the db update query, whereas `null` doesn't :+1:

Reply:
Ah ofcourse - was thinking it'd default to null, undefined makes sense.